src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -818,12 +818,9 @@ public function parseTable($table)
     {
         [$connection, $table] = Str::contains($table, '.') ? explode('.', $table, 2) : [null, $table];
 
-        if (Str::contains($table, '\\') && class_exists($table)) {
+        if (Str::contains($table, '\\') && class_exists($table) && is_a($table, Model::class, true)) {
             $model = new $table;
-
-            if ($model instanceof Model) {
-                $table = $model->getTable();
-            }
+            $table = $model->getTable();
```

Comment:
This line should just become `$table = (new $table)->getTable();`, and the line above deleted.

Reply:
I don't think you can use instanceof on strings